packages/test/test-end-to-end-tests/src/test/messageSize.spec.ts

```diff
@@ -390,7 +390,7 @@ describeCompat("Message size", "NoCompat", (getTestObjectProvider) => {
 					).timeout(chunkingBatchesTimeoutMs);
 				}));
 
-			itExpects(
+			itExpects.skip(
```

Comment:
Can we add a comment explaining the situation, and note the work item (in the form `AB#12345`) tracking re-enabling it?

Reply:
That's a good idea. I will add more details. Thanks!